Configuration/StandardSequences/python/VtxSmeared.py

```diff
@@ -58,7 +58,7 @@
     'Realistic100ns13TeVCollisionBetaStar90mLowBunches' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic100ns13TeVCollisionBetaStar90mLowBunches_cfi',
     'Realistic25ns13TeVEarly2017Collision' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic25ns13TeVEarly2017Collision_cfi',
     'Realistic25ns13TeVEarly2018Collision' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic25ns13TeVEarly2018Collision_cfi',
-
+    'RealisticPbPbCollision2018' : 'IOMC.EventVertexGenerators.VtxSmearedRealisticPbPbCollision2018_cfi',
 }
 VtxSmearedDefaultKey='Realistic50ns13TeVCollision'
-VtxSmearedHIDefaultKey='RealisticHICollision2015'
+VtxSmearedHIDefaultKey='RealisticPbPbCollision2018'
```

Comment:
@ColeLindsey @civanch this was not updated in master, why? I think we would like to have 104X and 103X consistent, right?

Reply:
@fabiocos Yes, you're right. Updating this default value was mentioned in an email that I didn't see until after I'd done the last update for the PR for master. The same email mentioned that it was rarely used so I didn't want to risk delaying that PR any longer. I did it on this backport and then figured I could do it on the master later. I hope that's okay.